[BE][1/2] Move original_weights_lookup attribute to constant

bbeckca opened the pull request:
Summary: As title. Cleaning usages by using global constant.

Test Plan: `buck test 'fbcode//mode/opt' fbcode//caffe2/test:quantization_fx -- --exact 'caffe2/test:quantization_fx - test_keep_original_weights (quantization.fx.test_quantize_fx.TestQuantizeFx)'`

Differential Revision: D72892815




cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv

Skylion007 commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -530,7 +532,7 @@ def load_arg(a):
 
     if keep_original_weights:
         setattr(  # noqa: B010
```
Does this satisfy B010 now?

bbeckca replied:
Not sure. If this is a flake8 warning, then it might not be satisfied as it's specific to property access.
https://fburl.com/zo2vrj32
<img width="547" alt="Screenshot 2025-04-14 at 12 55 48 PM" src="https://github.com/user-attachments/assets/b75defd0-7a50-4216-af1a-8ae7112d1df6" />

Commit:
[BE][1/2] Move original_weights_lookup attribute to constant (#151241)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/151241

As title. Cleaning usages by using global constant.

Test Plan: `buck test 'fbcode//mode/opt' fbcode//caffe2/test:quantization_fx -- --exact 'caffe2/test:quantization_fx - test_keep_original_weights (quantization.fx.test_quantize_fx.TestQuantizeFx)'`

Differential Revision: D72892815

hl475 reviewed (approved):
Thanks!
